mozwebqa.cfg

```diff
@@ -1,4 +0,0 @@
-[DEFAULT]
-api = webdriver
-baseurl = https://oneanddone-dev.allizom.org
-tags = oneanddone-tests
```

Comment:
Could you restore this by adding a `conftest.py` that sets the tags to include `oneanddone` (the -tests suffix is unnecessary).